packages/rocketchat-lib/server/models/Messages.js

```diff
@@ -19,6 +19,14 @@ RocketChat.models.Messages = new class extends RocketChat.models._Base {
 	}
 
 	// FIND
+	findOneById(_id, options) {
```

Comment:
All models already have the `findOneById` there is no necessary to reimplement here